### PR TITLE
🚀 Release v1.7.5

### PR DESCRIPTION
**Pull request by rajankumary2k**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by adilGhaffarDev** [comment]
this PR also has 1.8.0 notes can you delete those

**Comment by rajankumary2k** [comment]
Deleted.

**Review by Sunnatillo — COMMENTED** [review]
LGTM

**Comment by sbueringer** [comment]
/hold

@rajankumary2k Please squash this PR so it only contains one commit that adds the new v1.7.5. md file

**Comment by rajankumary2k** [comment]
Okay. Will do

File: CHANGELOG/v1.7.5.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+## Changes since v1.7.4
+## :chart_with_upwards_trend: Overview
+- 20 new commits merged
+- 1 feature addition ✨
+- 6 bugs fixed 🐛
+
+## :sparkles: New Features
+- ClusterCacheTracker: Add QPS & burst options & flags for ClusterCacheTracker (#10894)
+
+## :bug: Bug Fixes
```
Comment (sbueringer) [inline CHANGELOG/v1.7.5.md:17]: Pleas also include https://github.com/kubernetes-sigs/cluster-api/pull/11043/files here

It will merge in a few minutes

Comment (rajankumary2k) [reply CHANGELOG/v1.7.5.md:17]: Done.

**Comment by adilGhaffarDev** [comment]
LGTM from my side
/approve
/ok-to-test
I will un-hold it half an hour before the release cut meeting.

**Review by chandankumar4 — APPROVED** [review]
lgtm

**Comment by sbueringer** [comment]
Thx!

/lgtm
/approve

**Pull request merged**